website/source/docs/platform/k8s/mini.html.md

```diff
@@ -0,0 +1,324 @@
+---
```

Comment:
should be `minikube.html.md`

Reply:
Isn't there a minikube guide? I think it could link to that. I don't see why we'd want to duplicate the content from the run.html page